### ISSUE TITLE
Search Result Podcast Date (when background green): Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by davefollett** [codingblocks/podcast-app#233]
## 💥 Impact rating
serious

## 👣 Steps to reproduce
Add a podcast to the queue, start playing it, pause it, and then [run the aXe Accessibility Test](https://github.com/codingblocks/podcast-app/wiki/Accessibility-(a11y)-Testing#run-axe-accessibility-test).

## ⚛️ Element(s) with issue
Element location
`.hTfpdO > section > b > span`

Element source
`<span>(2018-08-21)</span>`

## 🛠️ Recommended fix
Fix the following:
Element has insufficient color contrast of 2.79 (foreground color: #157169, background color: #7fbfb8, font size: 9.6pt, font weight: bold). Expected contrast ratio of 4.5:1

Related node:
`.hTfpdO`

## 📚 Reference information
https://dequeuniversity.com/rules/axe/3.0/color-contrast?application=AxeChrome

**Comment by davefollett** [comment]
Closed by #237
